lib/money/src/Conversions.cpp

```diff
@@ -0,0 +1,36 @@
+#include <cmath>
+
+#include "Conversions.h"
+
+namespace {
+	struct Coefficient {
+		static constexpr double USD_USC = { 100 };
+		static constexpr double USC_USD = { 1 / USD_USC };
+		static constexpr double USD_DKK = { 1.5 };
+		static constexpr double DKK_USD = { 1 / USD_DKK };
+		static constexpr double DKK_USC = { DKK_USD * USD_USC };
+		static constexpr double USC_DKK = { USC_USD * USD_DKK };
+	} coef;
```

Comment:
You don't need to create an instance of the Coefficient class.
You can use direct access to the static members
 Coefficient::USD_USC;

Or it'd be better to use just constants without a struct/class introduction.
namespace {
    constexpr double USD_USC = { 100 };
    constexpr double USC_USD = { 1 / USD_USC };
   ....
}